assets/src/components/editing/editor/normalizers/normalizer.ts

```diff
@@ -91,14 +91,14 @@ export function installNormalizer(
         }
       }
 
-      options.spacesNormalize && spacesNormalize(editor, node, path);
-      options.blockNormalize && blockNormalize(editor, node, path);
-      options.codeNormalize && codeNormalize(editor, node, path);
-      options.listNormalize && listNormalize(editor, node, path);
-      options.tableNormalize && tableNormalize(editor, node, path);
+      if (options.spacesNormalize && spacesNormalize(editor, node, path)) return;
```

Comment:
These if/return bits are where we return early if a normalization change was made so a new loop starts.